user_tools/src/spark_rapids_tools/storagelib/csppath.py

```diff
@@ -274,6 +277,12 @@ def __init__(
     def __str__(self) -> str:
         return self._fpath
 
+    @classmethod
+    def init_env_vars(cls, path: Union[str, 'CspPath']) -> None:
+        if cls.protocol_prefix == AZURE_STORAGE_PREFIX:
+            account_name = get_abfs_account_name(str(path))
+            os.environ['AZURE_STORAGE_ACCOUNT_NAME'] = account_name
+
```

Comment:
This is very specific to `AdlsPath` class which extends `CspPath`. It is better to be inside `adlspath.py`.
I did not look deeper in the code, but my intuition is that this code block should be inside `adlspath.py` by overriding `is_protocol_prefix`

Reply:
Moved this to `adlspath.py`, thanks!